openedx/features/caliper_tracking/tests/expected/edx.special_exam.practice.attempt.created.json

```diff
@@ -0,0 +1,53 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "NavigatedTo",
+  "actor": {
+    "id": "http://localhost:18000/u/edx",
+    "name": "edx",
+    "type": "Person"
+  },
+  "eventTime": "2018-10-24T10:25:19.659Z",
+  "extensions": {
+    "extra_fields": {
+      "accept_language": "en-US,en;q=0.9",
+      "agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.99 Safari/537.36",
+      "course_id": "course-v1:UCP+CS4230+PF_2018",
+      "event_source": "server",
+      "event_type": "edx.special_exam.practice.attempt.created",
+      "host": "e5d2c598af1a",
+      "ip": "172.18.0.1",
+      "org_id": "UCP",
+      "page": null,
+      "path": "/api/edx_proctoring/v1/proctored_exam/attempt",
+      "session": "83862e716c03cd4729b4dfc3dd73390c",
+      "user_id": 2
+    }
+  },
+  "id": "urn:uuid:cd088ca7-c044-405c-bb41-0b2a8506f907",
+  "object": {
+    "extensions": {
+      "attempt_allowed_time_limit_mins": null,
+      "attempt_code": "656070CB-D5D9-46F0-A7DD-CFC3FF701F33",
+      "attempt_completed_at": null,
+      "attempt_event_elapsed_time_secs": null,
+      "attempt_id": 4,
+      "attempt_started_at": null,
+      "attempt_status": "created",
+      "attempt_user_id": 2,
+      "exam_content_id": "block-v1:UCP+CS4230+PF_2018+type@sequential+block@f4d9413d3bc14a419b7de32b90431019",
+      "exam_default_time_limit_mins": 30,
+      "exam_id": 5,
+      "exam_is_active": true,
+      "exam_is_practice_exam": true,
+      "exam_is_proctored": true
+    },
+    "id": "http://localhost:18000/courses/course-v1:UCP+CS4230+PF_2018/courseware/360ca77f129246b2b495e40ade55c066/f4d9413d3bc14a419b7de32b90431019/1?activate_block_id=block-v1%3AUCP%2BCS4230%2BPF_2018%2Btype%40vertical%2Bblock%404c6968a3bc2441ba8f38e5e80319de2c",
+    "name": "Practice",
+    "type": "WebPage"
+  },
+  "referrer": {
+    "id": "http://localhost:18000/courses/course-v1:UCP+CS4230+PF_2018/courseware/360ca77f129246b2b495e40ade55c066/f4d9413d3bc14a419b7de32b90431019/1?activate_block_id=block-v1%3AUCP%2BCS4230%2BPF_2018%2Btype%40vertical%2Bblock%404c6968a3bc2441ba8f38e5e80319de2c",
+    "type": "WebPage"
+  },
+  "type": "NavigationEvent"
```

Comment:
Why `NavigationEvent`?

Reply:
Because when we click on the `Submit` button then this event generates and the control goes to the next page where it has been asked `Yes, I am sure to submit`. So that's why I made it `NavigationEvent`

Reply:
Is there any other event that is generated when the user actually submits?